documentation/installation.md

```diff
@@ -10,24 +10,35 @@ system.
 
 Vagrant uses [VirtualBox](https://www.virtualbox.org/) to create isolated
 virtual machines with the developer environment set up. To provision and
-configure the developer environment as needed Vagrant uses
+configure the developer environment as needed, Vagrant uses
 [Ansible](https://www.ansible.com/).
 
-You will need to install all these tools before you can set up your development
+First, make sure that Intel virtualization is enabled in your BIOS / UEFI firmware.
```

Comment:
@alanoe: nit: this cannot always be Intel virtualization as AMD CPUs, Ryzen series in particular, are very popular. So it might be better to explicitly mention `Intel VT-x/AMD-V virtualization hardware extensions' instead.

```suggestion
First, make sure that Intel VT-x/AMD-v virtualization hardware extension are enabled in your BIOS / UEFI firmware.
```

Reply:
Done